R/scale-.R

```diff
@@ -633,6 +547,29 @@ check_breaks_labels <- function(breaks, labels, call = NULL) {
   TRUE
 }
 
+check_update_params <- function(scale, params) {
+  if (inherits(scale, "ScaleContinuous")) {
+    args <- fn_fmls_names(continuous_scale)
+  } else if (inherits(scale, "ScaleDiscrete")) {
+    args <- fn_fmls_names(discrete_scale)
+  } else if (inherits(scale, "ScaleBinned")) {
+    args <- fn_fmls_names(binned_scale)
```

Comment:
Would be nice if this information was somehow incorporated into the class, rather than having to check an "unlinked" constructor function